### DISCUSSION TITLE
tasks/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#16241] [Docs]
# tasks/

Explore Ultralytics YOLOv8 for detection, segmentation, classification, OBB, and pose estimation with high accuracy and speed. Learn how to apply each task.

https://docs.ultralytics.com/tasks/

**Comment by hrbjoern** [comment]
Would be nice to have examples for _tracking_, too ...

**Reply by glenn-jocher** [reply]
@hrbjoern thank you for your suggestion! We currently have tracking examples in our documentation. You can explore them for more insights on implementing tracking with YOLOv8.

**Reply by hrbjoern** [reply]
OK, thanks. I just figured that since it's mentioned in the beginning of the page (photo), it should also be explained ...

**Reply by glenn-jocher** [reply]
Yes sorry for the confusion. We've placed the Tracking docs under Modes rather than Tasks here: https://docs.ultralytics.com/modes/track/

**Comment by NasserAlqeifi** [comment]
Hi there! Could anyone point me to the papers for YOLOv8 and YOLOv11 Instance Segmentation? Thank you!

**Reply by glenn-jocher** [reply]
@NasserAlqeifi hello! The YOLOv8 and YOLOv11 models don't have specific papers yet. For detailed information, please refer to the [Ultralytics documentation](https://docs.ultralytics.com/tasks/). If you have further questions, feel free to ask!

**Reply by NasserAlqeifi** [reply]
Hi @glenn-jocher ,

Thank you for the information! I'm a master's student currently using the YOLOv8 Instance Segmentation model for my thesis. I've also tried using YOLOv11 and noticed that the documentation mentions it utilizes a variant of the U-Net architecture for segmentation.

I would love to know more about how segmentation and object detection work in YOLOv11, as well as the key differences between YOLOv8 and YOLOv11 in these tasks. Any additional insights would be greatly appreciated!

Thank you!

**Reply by glenn-jocher** [reply]
Thank you for your interest! YOLOv11 uses a variant of the U-Net architecture for segmentation, enhancing accuracy and speed. The key difference from YOLOv8 is the architectural improvements for better performance across tasks. For more details, please refer to the [Ultralytics documentation](https://docs.ultralytics.com/tasks/).

**Comment by DukeZacharia** [comment]
Good, Afternoon, does Yolo work for Human Action Detection in realtime?

**Reply by glenn-jocher** [reply]
@DukeZacharia good afternoon! YOLO can be used for real-time human action detection through pose estimation. For more details, please explore the pose estimation section in the Ultralytics documentation.

**Comment by AashishThakur1321** [comment]
how can I provide different object ids in videos to same types of objects like there are 5 persons in a video frame, then how to provide different ids to these persons such that it tracks these objects with same id throughout the videos

**Reply by glenn-jocher** [reply]
@AashishThakur1321 to track multiple persons with unique IDs in videos using YOLO11, utilize the built-in tracking mode with `tracker=bot-sort.yaml` or `tracker=bytetrack.yaml` configuration. Example CLI command:  
`yolo track model=yolo11n.pt source=video.mp4`  
This automatically assigns unique IDs to each person and maintains them across frames. For customization, see our [tracking documentation](https://docs.ultralytics.com/modes/track/).

**Comment by Saimul-Shanto** [comment]
Can I use XAI with YOLOv11 or YOLOv9 ? If so, how can I apply it to an object detection task?

**Comment by DukeZacharia** [comment]
Yes, you can utilize Explainable AI (XAI) with YOLOv9 or YOLOv11,

even though these versions are unofficial or experimental extensions beyond
YOLOv8.

YOLO is fundamentally a convolutional neural network (CNN),

which means that many XAI techniques applicable to CNNs or object detection
models can be adapted for use with these versions.

**Comment by adhinajay** [comment]
I quantized a YOLOv8 model. Now I want to fine tune it, so how can I convert the ONNX file back to a .pt file?
